packages/my-react-package/src/MetaContract.tsx

```diff
@@ -27,11 +28,18 @@ function enableNoSuchMethod(obj: MetaContract) {
 class MetaContract {
 
     // Store different chains' contracts information
-    private supportedChains = ["ethereum", "polygon"] as Array<string>; 
-    contract = {} as ContractsJson;
+     supportedChains =
```

Comment:
These values are already there in the constants/supportedchains files. Please reuse that here